internal/main.go

```diff
@@ -18,6 +19,16 @@ func main() {
 		panic("No TMDB key provided")
 	}
 
+	log.SetFlags(log.Ldate | log.Ltime | log.Lshortfile)
+
+	// permission mode 0666 -> all users can read and write but cannot execute file
+	logFile, err := os.OpenFile("logs.txt", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+	if err != nil {
+		log.Println("there was an error with logs.txt: ", err)
+	}
+
+	log.SetOutput(logFile)
```

Comment:
Is it possible to have multiple outputs? It seems like it isn't logging to the terminal anymore

Reply:
Also, should the file be closed (maybe with defer)?

Reply:
Done

Reply:
Sick